packages/react-docs/UPGRADE-GUIDE.md

```diff
@@ -0,0 +1,264 @@
+- **ModalFooter:**
+  - Removed prop `isLeftAligned` [(#4017)](https://github.com/patternfly/patternfly-react/pull/4017)
+
+### Nav
+- **Nav:** 
+  - Changed default theme to dark. Use `theme="light"` if you wish to use the light variant. [(#4116)](https://github.com/patternfly/patternfly-react/pull/4116)
+- **NavList:** 
+  - Removed prop `variant`. Pass `variant="horizontal"` or `variant="tertiary"` to `Nav` instead. [(#4225)](https://github.com/patternfly/patternfly-react/pull/4225)
+
+### Options menu
+- **OptionsMenu:**
+  - Removed prop `ariaLabelMenu`.
+- **OptionsMenuItemGroup:**
+  - Removed prop `ariaLabel` in favor of `aria-label`
+- **OptionsMenuToggleWithText**:
+  - Removed prop `ariaHasPopup` in favor of `aria-haspopup`
+  - Removed props `isFocused` and `isHovered`, all instances of them should be removed from your application. [(#4116)](https://github.com/patternfly/patternfly-react/pull/4116)
+- **OptionsMenuToggle:** 
+  - Removed props `isFocused` and `isHovered`, all instances of them should be removed from your application. [(#4116)](https://github.com/patternfly/patternfly-react/pull/4116)
+
+### Page
+- **Page:**
+  - Made mainContainerId a required property for skip to content component [(#3904)](https://github.com/patternfly/patternfly-react/pull/3904)
+  - Changed default theme to dark. Use `theme="light"` if you wish to use the light variant. [(#4116)](https://github.com/patternfly/patternfly-react/pull/4116)
+  - Renamed `noPadding` to `hasNoPadding`. [(#4133)](https://github.com/patternfly/patternfly-react/pull/4133)
+- **PageSection:**
+  - Removed prop `noPaddingMobile` from PageSection in favor of `hasNoPaddingOn={PageSectionBreakpoints[]}` [(#4133)](https://github.com/patternfly/patternfly-react/pull/4133)
+  - Added `hasPaddingOn` and `hasNoPaddingOn` properties to PageSection, accounting for page size breakpoints. Breakpoints are defined in the `PageSectionBreakpoints` enum. [(#4133)](https://github.com/patternfly/patternfly-react/pull/4133)
+- **PageHeader**:
+  - Removed prop `avatar`. Add the avatar to the `PageHeaderTools` component instead (which is passed into `PageHeader` via the `headerTools` prop. [(#4223)](https://github.com/patternfly/patternfly-react/pull/4223)
+  - Removed prop `toolbar`. Use the `headerTools` prop instead. Also, if you previously used the `Toolbar`, `ToolbarGroup`, or `ToolbarItem` components for the header, replace them with the `PageHeaderTools`, `PageHeaderToolsGroup`, and `PageHeaderToolsItem` components. [(#4223)](https://github.com/patternfly/patternfly-react/pull/4223)
+
+### Pagination
+- Removed obsolete 'left' and 'right' variants. These should be replaced with 'top' (default) or 'bottom'. [(#4202)](https://github.com/patternfly/patternfly-react/pull/4202)
+
+### Popover
+- **PopoverHeader:** 
+  - Title now has a default size of medium instead of extra large [(#4030)](https://github.com/patternfly/patternfly-react/pull/4030)
```

Comment:
```suggestion
  - Title now has a default size of medium instead of extra large. [(#4030)](https://github.com/patternfly/patternfly-react/pull/4030)
```

Reply:
We're pretty inconsistent with the periods. When should we use/not use them in bullet points?

Reply:
I noticed that we were using periods for the most part when something was a full sentence, so that's why I added one here. As a general rule, periods should be used in bullet lists if the list item is a full sentence or if the list item creates a full sentence when combined with the clause introducing the list. With that said though, I don't think we need to get too into perfecting it for this release since we're under a time crunch! We can just keep it in mind for next time.